### ISSUE TITLE
what is it related to? 

### ISSUE THREAD

**Issue by RostyslavPaliuha** [docker-library/openjdk#184]
Hello, please help me, if I run my services in IDE or up docker containers without SBA all works fine. When I added SBA server/client dependencies and configured and ran in IntelliJ on windows 10 it worked fine, but don`t work properly in docker. I try change images from OpenJDK: alpine to OpenJDK:8u162-jre-stretch and nothing changed.
I found that this error related with hostname. Is it the solution to change it to localhost?
ERROR

config_1 | 2018-04-12 10:29:41.918 ERROR 1 --- [gistrationTask1] o.s.s.s.TaskUtils$LoggingErrorHandler : Unexpec ted error occurred in scheduled task. config_1 | config_1 | java.lang.IllegalArgumentException: linuxkit-00155d002420: linuxkit-00155d002420: Name or service not kn own config_1 | at de.codecentric.boot.admin.client.registration.DefaultApplicationFactory.getLocalHost(DefaultAppli cationFactory.java:183) ~[spring-boot-admin-client-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT] config_1 | at de.codecentric.boot.admin.client.registration.DefaultApplicationFactory.getServiceHost(DefaultApp licationFactory.java:166) ~[spring-boot-admin-client-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT] config_1 | at de.codecentric.boot.admin.client.registration.DefaultApplicationFactory.getServiceBaseUrl(Default ApplicationFactory.java:103) ~[spring-boot-admin-client-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT] config_1 | at de.codecentric.boot.admin.client.registration.ServletApplicationFactory.getServiceUrl(ServletAppl icationFactory.java:58) ~[spring-boot-admin-client-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT] config_1 | at de.codecentric.boot.admin.client.registration.ServletApplicationFactory.getManagementBaseUrl(Serv letApplicationFactory.java:73) ~[spring-boot-admin-client-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT] config_1 | at de.codecentric.boot.admin.client.registration.DefaultApplicationFactory.getHealthUrl(DefaultAppli cationFactory.java:150) ~[spring-boot-admin-client-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT] config_1 | at de.codecentric.boot.admin.client.registration.DefaultApplicationFactory.createApplication(Default ApplicationFactory.java:71) ~[spring-boot-admin-client-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT] config_1 | at de.codecentric.boot.admin.client.registration.ApplicationRegistrator.createApplication(Applicatio nRegistrator.java:154) ~[spring-boot-admin-client-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT] config_1 | at de.codecentric.boot.admin.client.registration.ApplicationRegistrator.register(ApplicationRegistra tor.java:71) ~[spring-boot-admin-client-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT] config_1 | at org.springframework.scheduling.support.DelegatingErrorHandlingRunnable.run(DelegatingErrorHandlin gRunnable.java:54) ~[spring-context-5.0.5.RELEASE.jar!/:5.0.5.RELEASE] config_1 | at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [na:1.8.0_162] config_1 | at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) [na:1.8.0_162] config_1 | at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPo olExecutor.java:180) [na:1.8.0_162] config_1 | at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecu tor.java:294) [na:1.8.0_162] config_1 | at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_162] config_1 | at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_162] config_1 | at java.lang.Thread.run(Thread.java:748) [na:1.8.0_162] config_1 | Caused by: java.net.UnknownHostException: linuxkit-00155d002420: linuxkit-00155d002420: Name or service not known config_1 | at java.net.InetAddress.getLocalHost(InetAddress.java:1505) ~[na:1.8.0_162] config_1 | at de.codecentric.boot.admin.client.registration.DefaultApplicationFactory.getLocalHost(DefaultAppli cationFactory.java:181) ~[spring-boot-admin-client-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT] config_1 | ... 16 common frames omitted config_1 | Caused by: java.net.UnknownHostException: linuxkit-00155d002420: Name or service not known config_1 | at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method) ~[na:1.8.0_162] config_1 | at java.net.InetAddress$2.lookupAllHostAddr(InetAddress.java:928) ~[na:1.8.0_162] config_1 | at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1323) ~[na:1.8.0_162] config_1 | at java.net.InetAddress.getLocalHost(InetAddress.java:1500) ~[na:1.8.0_162] config_1 | ... 17 common frames omitted

**Comment by tianon** [comment]
It's a little hard to parse that long block of unformatted text, but it looks like you've got some code in your application which is trying to parse `config_1` as a hostname and is failing to look it up.  Have you set an appropriate `hostname` on your service/container?  Have you added a `network-alias` to your container so that DNS resolves `config_1` appropriately?

There's not really enough detail here for someone to help you figure out your issue.  What I'd recommend is that you gather togethre more details about your specific issue (especially a simple way for someone else to reproduce it), then post it to [the Docker Community Forums](https://forums.docker.com/), [the Docker Community Slack](https://blog.docker.com/2016/11/introducing-docker-community-directory-docker-community-slack/), or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=docker).
